bundles/org.openhab.binding.mqtt.fpp/src/main/feature/feature.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.mqtt.espmilighthub-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Comment:
espmilighthub seems like a copy/paste error, change to fpp. Some more in this file.

Reply:
fixed